util/run_intg_tests.py

```diff
@@ -205,7 +205,7 @@ def test_config(sim_conf, path_root, logpath, interpreter):
         logging.error("Back-end still running after requesting to stop")
         backend.kill()
 
-    if gui and gui.returncode != 143:  # SIGTERM return code
+    if gui and gui.returncode != -15:  # SIGTERM return code
```

Comment:
The point here is that when we call "odemis-stop" after 10s, the GUI ends with a error code, and we don't want to catch it as an "Error". At least with Python 2, in practice it's 143. You can try by starting the GUI from the command-line (`odemis-gui`), and later call `odemis-stop`. Use `echo $?` to check the return code.

Reply:
I don't get it. When I print the returncode inside the script, I get -15 and the GUI starts fine. Then I try to reproduce it in a python shell and the return value is 143. How is this possible? The 2 numbers might be related, 15 seems to be signal.SIGTERM value, and it seems that sometimes it's added to 128 to produce the final error code (=143).

Reply:
All testcases that pass return this `ERROR:root:GUI failed to start, with return code -15`